### ISSUE TITLE
Grid with checkboxes in first column navigates through the checkboxes only.

### ISSUE THREAD

**Issue by martintabakov** [telerik/kendo-ui-core#6238]
### Bug report

In a Grid with checkboxes in the first column, if you navigate through the Grid for a second time, only the checkboxes are focused.

### Reproduction of the problem
1. Open this Dojo example - https://dojo.telerik.com/@martin.tabakov@progress.com/EZERotAN/5.
2. Navigate through the whole Grid with tab.
3. Navigate a second time through the Grid.

### Current behavior
When navigating through the Grid for a second time, only the checkboxes are focused.

### Expected/desired behavior
All the cells of the Grid should be focused.

### Environment

* **Kendo UI version:** 2020.3.1118
* **Browser:** [all]

**Comment by ag-petrov** [comment]
The descriptions has a small mistake, the second time the grid table is focused, the focus order the second time is header checkbox, grid table, checkboxes in select column. When the grid table is focused the last active cell which was used when navigating the last time becomes active. The way the elements are focused is the way how HTML works and the part where the last active cell becomes active again is design behavior of the grid.

**Comment by ag-petrov** [comment]
If for some reason someone wants the iteration to start all over again from scratch he will have to clear the cached last focused cell in the grid as shown in [this ](https://dojo.telerik.com/uSUJafeD/2)dojo. Please refer to the focusout handler logic. 

```
}).data("kendoGrid").wrapper.on("focusout", "table", function () {
        $("#kendo").getKendoGrid()._current = null;
      })
```
